registrations/api.py

```diff
@@ -166,7 +164,7 @@ def send_message(self, request, pk=None, version=None):
         try:
             send_mass_html_mail(messages, fail_silently=False)
         except SMTPException as e:
-            logger.exception("Couldn't send mass HTML email.")
+            capture_exception(e)
```

Comment:
Since we have set up sentry logging in `settings.py`, using `logger.exception` should be enough to send errors to sentry.

Reply:
Yes, that seems to be the default behavior of the Sentry SDK: https://docs.sentry.io/platforms/python/guides/logging/. Learned something new here :-) Now changed to using `logger.exception`